packages/inventory-compliance/src/ComplianceRemediationButton.js

```diff
@@ -53,9 +57,17 @@ class ComplianceRemediationButton extends React.Component {
         });
     }
 
-    noRemediationAvailable = () => {
+    remediationAvailable = () => {
```

Comment:
I found this logic way more confusing for the negative case :) It's confusing even for the positive case.